src/components/TextContainers.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+export function Header1({ children, ...rest }) {
+  return (
+    <Typography component="h1" variant="h4" gutterBottom {...rest}>
+      {children}
+    </Typography>
+  );
+}
+
+export function Header2({ children, ...rest }) {
+  return (
+    <Typography component="h2" variant="h5" gutterBottom {...rest}>
+      {children}
+    </Typography>
+  );
+}
+
+export function Header3({ children, ...rest }) {
+  return (
+    <Typography component="h3" variant="h6" gutterBottom {...rest}>
+      {children}
+    </Typography>
+  );
+}
+
+export function Header4({ children, ...rest }) {
+  return (
+    <Typography
+      component="h4"
+      variant="subtitle2"
+      gutterBottom
+      style={{ fontStyle: 'italic', fontWeight: 'bold' }}
+      {...rest}
+    >
+      {children}
+    </Typography>
+  );
+}
+
+export function Paragraph({ children, ...rest }) {
+  return (
+    <Typography paragraph {...rest}>
+      {children}
+    </Typography>
+  );
+}
+
+export function ListItem({ children, ...rest }) {
```

Comment:
I think ListItem is a material UI component we already use in quite a few places. I'd support a re-name of this one... dunno what I'd change it to, though. Litem lmao?